llvm/lib/CAS/MappedFileRegionBumpPtr.cpp

```diff
@@ -218,6 +233,10 @@ void MappedFileRegionBumpPtr::initializeBumpPtr(int64_t BumpPtrOffset) {
   if (!BumpPtr->compare_exchange_strong(ExistingValue, BumpPtrEndOffset))
     assert(ExistingValue >= BumpPtrEndOffset &&
            "Expected 0, or past the end of the BumpPtr itself");
+
+  if (Logger)
+    Logger->log_MappedFileRegionBumpPtr_create(Path, *FD, data(), capacity(),
```

Comment:
Not sure if `data()` is useful since this is the only pointer recorded. I guess it might be useful when reading together with crash log?

Reply:
This is useful for matching the mapped file up with later operations like allocations, subtrie creation and record creation that all log the region's base pointer as well. Otherwise you don't know whether a subtrie is in the index or in the action cache, for example.